src/NosCore.Controllers/CommandPacketController.cs

```diff
@@ -53,6 +68,110 @@ public void Shout(ShoutPacket shoutPacket)
             ServerManager.Instance.BroadcastPackets(new List<PostedPacket>(new[] {sayPostedPacket, msgPostedPacket}));
         }
 
+        [UsedImplicitly]
+        public void CreateItem(CreateItemPacket createItemPacket)
```

Comment:
A lot of unnecessary nested conditions in this method

Reply:
Yes but admin command so not something really usefull